mesa_geo/raster_layers.py

```diff
@@ -415,6 +460,27 @@ def from_file(
             obj.apply_raster(values, attr_name=attr_name)
             return obj
 
+    def to_file(self, raster_file, attr_name: str = None, driver="GTiff"):
```

Comment:
Same here. Also, `raster_file` is not yet annotated.

Reply:
Updated too.